app/javascript/packages/memorable-date/index.js

```diff
@@ -0,0 +1,83 @@
+import { t } from '@18f/identity-i18n';
```

Comment:
Since we are [TypeScript-first](https://github.com/18F/identity-idp/blob/main/docs/frontend.md#at-a-glance-1) for new files, can we make this a `.ts` file instead?